packer_test/common/commands.go

```diff
@@ -66,8 +66,14 @@ func (pc *packerCommand) SetWD(dir string) *packerCommand {
 }
 
 // UsePluginDir sets the plugin directory in the environment to `dir`
-func (pc *packerCommand) UsePluginDir(dir string) *packerCommand {
-	return pc.AddEnv("PACKER_PLUGIN_PATH", dir)
+func (pc *packerCommand) UsePluginDir(dir *PluginDirSpec) *packerCommand {
+	return pc.UseRawPluginDir(dir.dirPath)
+}
+
+// UseRawPluginDir is meant to be used for setting the plugin directory with a
+// raw directory path instead of a PluginDirSpec.
```

Comment:
I believe this is overrides the plugin command for a single invocation correct? 
```suggestion
// UseRawPluginDir can be used for setting a plugin directory to a
// raw directory path instead of a PluginDirSpec.
// The temporary plugin path is set for a single command invocation. 
```

Reply:
It is indeed for one invocation, that said, all those options are, as `PackerCommand` is meant to be essentially discarded after `Assert`, so pointing the ephemeral nature of the call here and not elsewhere may be misleading I'm afraid

Reply:
I see. Okay no need for the call out then.